app/presenters/keyword_presenter.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class KeywordPresenter
```

Comment:
@aetheryna Should we add test for this presenter?

Reply:
Definitely, added here https://github.com/aetheryna/google-scraper-ruby/pull/34/commits/f4ffa13498e6dba8963a0094691668859c6dd849 🧪